src/omv/imlib/bayer.c

```diff
@@ -10,7 +10,7 @@
  */
 #include "imlib.h"
 
-void imlib_debayer_line_to_binary(int x_start, int x_end, int y_row, uint32_t *dst_row_ptr, image_t *src)
```

Comment:
IMHO the multiple small functions were much more readable and easier to add to than 1 monolithic function. At least keep the smaller functions internally (inline'd static) and call them from debayer_line/image, and add a function/macro to clip the row pointers as needed.

Reply:
The amount of code generated would be enormous. I merged it all into 1 method so we wouldn't have 4K lines of code here.

Reply:
Not looking for speed ups, but it will be harder to add to/fix in the future, your choice anyway.